SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -606,19 +607,25 @@ void ASpatialDebugger::DrawDebug(UCanvas* Canvas, APlayerController* /* Controll
 	}
 #endif
 
+	if (bSelectActor)
+	{
+		SelectActorToTag(Canvas);
+		return;
+	}
+	else
+	{
+		// Change mouse cursor back to normal
+		LocalPlayerController->bShowMouseCursor = true;
```

Comment:
Maybe add a comment here that this code assumes that we have the config UI open when going in and out of the actor select mode? Otherwise the assumption that the mouse cursor should be visible is not necessarily true (would depend on what the game had set previously).